source/sass/component/_modal.scss

```diff
@@ -119,7 +119,7 @@ $c-modal-backdrop-color: var(--c-modal-backdrop-color, $color-alpha) !default;
     .c-modal__close.c-button.c-button--md,
     .c-modal__close.c-button.c-button--sm {
         cursor: pointer;
-        color: $color-white;
+        color: $color-black;
```

Comment:
Does this work with modals without conatiner? For example the search modal?

Reply:
Ill have to check more usage areas for the modal. Ill remove this code since there is another task open to fix this.